src/stores/order.store.spec.ts

```diff
@@ -23,60 +23,61 @@ const order: OrderWithTrades = {
 }
 
 describe('OrderStore', () => {
+  const storeOpts = { exchange: 'test', symbol: 'test', strategy: 'test' }
   let orderStore: OrderStore
 
   beforeEach(() => {
-    orderStore = new OrderStore()
-    orderStore.addSymbol('test', 'test', 'test')
+    orderStore = OrderStore.instance
```

Comment:
Same like for the other unit test, its a singleton and already constructed, so it should be enough to do this in a beforeAll. State isnt reseted so there should be no difference if i am right.

Can you check this for the other unit tests also? then i have not to copy the same comment multiple time.